game/src/sandbox/dashboards/commuter.rs

```diff
@@ -211,6 +211,15 @@ impl CommuterPatterns {
 
                 batch.push(Color::BLACK.alpha(0.5), base_block.shape.clone());
 
+                // Draw outline for Locked Selection
```

Comment:
existing issue that became obvious with this PR, we should draw the border beneath the inwar/outward icon, not above it.